certified-connectors/Data247logic/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3474 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Data247",
+    "description": "Welcome to the Data247 API! You can use our API to access the various data services we provide.",
+    "version": "1.0",
+    "contact":{
+		"name":"Marc Orenberg",
+		"url":"https://www.data247.com/docs/#introduction",
+		"email":"mohammad@logictree.com"
+	}
+  },
+  "host": "api.data247.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v3.0": {},
+    "/v3.0/MT/": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@Data247logic Please replace ALL the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
this is also done.